### DISCUSSION TITLE
skip checkup failure

### DISCUSSION THREAD

**Discussion by dsbowman** [skiptools/skip#293] [Q&A]
I am having issues finding assistance on the last two issues

[✓] Skip version 1.1.24 (= 1.1.24)
[✓] macOS version 15.2 (> 13.5.0)
[✓] macOS architecture: ARM
[✓] Swift version 6.0.2 (> 5.9.0)
[✓] Xcode version 16.1 (> 15.0.0)
[✓] Xcode tools SDKs: 5
[✓] Homebrew version 4.4.7 (> 4.1.0)
[✓] Gradle version 8.11.1 (> 8.6.0)
[✓] Java version 23.0.1 (> 21.0.0)
[✓] Android Debug Bridge version 1.0.41 (> 1.0.40)
[✓] Android Studio version: 2024.2
[✓] Android tools SDKs: 2
[✓] Resolve dependencies (4.06s)
[✓] Build hello-skip (12.18s)
[✓] Test Swift (7.85s)
[✓] Test Kotlin (16.78s)
[✗] Archive iOS ipa (3.78s)
[✗] The file “Payload” doesn’t exist.
[✗] Skip 1.1.24 checkup failed with 2 errors

**Comment by marcprux** [comment]
Can you look in the log that is referenced at the end of the command output for error messages?
